.github/workflows/release.yml

```diff
@@ -15,7 +15,7 @@ permissions:
 jobs:
   run-release:
     name: dxCompiler Release (Staging and Prod)
```

Comment:
I did not run/test this pipeline, but I ran into issues when running with 20.04 in different pipelines - so I started it on 24.04 which does not have sbt by default, so I added installation.

I've also updated libraries similarly to other workflows. Other workflows ran smoothly with these changes.